lib/LedgerSMB/Template/UI.pm

```diff
@@ -78,7 +78,7 @@ sub new_UI {
         if (!defined $engine) {
             $engine = Template->new(
                 ### TODO: These should be configurable absolute paths
-                INCLUDE_PATH => [ 'UI/', 'UI/lib/' ],
+                INCLUDE_PATH => [ 'UI/js', 'UI/', 'UI/lib/' ],
```

Comment:
Which line-change requires the extra `UI/js` path to be in there?

Reply:
Producing `UI/js/ui-header.html` from the `html-webpack-plugin` template, to be able to load it with the hashed JS & CSS. See https://github.com/ledgersmb/LedgerSMB/pull/4787/commits/167f34504071e4d6b1b267ba7ebac419ecadfba7#diff-11e9f7f953edc64ba14b0cc350ae7b9dR193
Compiled files, when exist, must be picked instead of the non-compiled sources.